RentACar/RentACar/CRUD/CarCRUD.cs

```diff
@@ -35,7 +35,11 @@ public Car Create(Car car)
             {
                 cars = DeserializeJsonToList(_jsonFile);
 
-                car.Id = cars.Last().Id + 1;
+                if ((cars.Count > 0))
+                    car.Id = (cars.Last().Id + 1);
```

Comment:
Are you sure that the car in the last position has the greater id?

Reply:
At first i think yes, because Create method always creates cars at the end of the file, and if i remove any elements in the beginning, middle or end of the file, the last car will always have the greater id.
**But** i'm thinking that maybe someone can inserts cars in another way. Is there something else that im not seeing?
Thank you very much!

Reply:
In this moment I can think in two possible solutions:
* Using [Guid](https://docs.microsoft.com/en-us/dotnet/api/system.guid?view=net-5.0) in place of integers.
* Look for the greater id and use it to be incremented. This can be done easily with Linq, something like: `var newId = cars.Map(x => x.Id).OrderByDesc(x => x).FirstOrDefault(0) + 1` (I did not verify this code)

Reply:
Oh nice! Matias told me something similar. I will take a look and get to work on it! 
When you use .Map method you mean .Select method? Or im wrong?
Thanks for your help!

Reply:
Yes, sorry, pseudocode 😛

Reply:
Wow our minds are synchronized like an Eva from NGE @andresmoschini

Reply:
> Wow our minds are synchronized like an Eva from NGE @andresmoschini

😱